R/plot_voronoi.R

```diff
@@ -39,18 +36,20 @@ plotvoronoi <- function(object, model = spc ~ x * y,
     stop("package latticeExtra is needed for Voronoi plots.")
   }
 
-  if (use.tripack) {
-    if (!requireNamespace("tripack")) {
-      stop("package tripack requested but not available.")
-    }
-  } else {
-    if (!requireNamespace("deldir")) {
-      stop("package deldir requested but not available.")
-    }
+  if (!(use.tripack == FALSE)) {
+    warning(paste0(
```

Comment:
`warning()` already `paste0()`s multiple arguments

Reply:
I fixed that